lib/google_scraping/dashboard/dashboard.ex

```diff
@@ -0,0 +1,61 @@
+defmodule GoogleScraping.Dashboard do
+  @moduledoc """
+  The Dashboard context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias GoogleScraping.Dashboard.Queries.KeywordQueries
+  alias GoogleScraping.Dashboard.Schemas.Keyword
+  alias GoogleScraping.Repo
+
+  @doc """
+  Returns the list of keywords associated with the given user.
+
+  ## Examples
+
+      iex> list_keywords(user_id)
+      [%Keyword{}, ...]
+
+  """
+  def list_keywords(user_id) do
+    Repo.all(KeywordQueries.user_keywords_query(user_id))
+  end
```

Comment:
This can be written as single line convention as per our convention https://nimblehq.co/compass/development/code-conventions/elixir/#functions
```suggestion
def list_keywords(user_id), do: Repo.all(KeywordQueries.user_keywords_query(user_id))
```